tests/fw_rules_tests/run_tests.py

```diff
@@ -12,7 +12,7 @@
     os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), '..', 'network-config-analyzer'))
 from nca import nca_main
 
-supported_formats = ['txt', 'yaml', 'csv', 'md']
+supported_formats = ['txt', 'yaml', 'csv', 'md', 'dot']
```

Comment:
```suggestion
supported_formats = ['txt', 'yaml', 'csv', 'md']
```

If we just add the dot format here, then the testing script will currently generate all new expected output files for this format when we run it locally..
We may need to consider adding support for this dot format in our tests in a separate issue, because we have some multiple items that are related:
- the output of the dot file should be sorted so that file-based comparison will not fail.
- update the script so that it will only check against existing expected output file, and will not auto generate new expected files if they are missing (maybe consider some flag for this).
- update the script to test variations of output configurations, not only through cli args.